tests/ui/checked_conversions.stderr

```diff
@@ -0,0 +1,52 @@
+error: Checked cast can be simplified: `u32::try_from`
+  --> $DIR/checked_conversions.rs:9:8
+   |
+LL |     if value <= (u32::max_value() as i64) && value >= 0 {
```

Comment:
Can you try creating a suggestion that will produce `u32::try_from(value).is_ok()`? You can change the test to be a rustfix test which will automatically check if the code still compiles after applying all suggestions.

Reply:
I added a new commit that should do exactly that - although I am not sure how to add a (potentially missing) include for std::convert::TryFrom.

Reply:
There's no need for an include, the compiler will tell the user what they need if they need it.

Reply:
But it is still needed for the rustfix test to work, right? Or am I missing something there, too?